scripts/mirror.js

```diff
@@ -302,71 +264,57 @@ const bumpChromeAndroid = (originalData, sourceData) => {
-    }
+  const chromeFalse =
+    sourceData.version_removed || sourceData.version_added === false;
+  const chromeNull = sourceData.version_added === null;
 
-    if (sourceData.notes) {
-      newData.notes = updateNotes(
-        sourceData.notes,
-        /Internet Explorer/g,
-        'Edge',
-      );
+  if (chromeFalse) {
+    if (
+      typeof sourceData.version_removed === 'string' &&
+      compareVersions.compare(sourceData.version_removed, '79', '<=')
+    ) {
+      // If this feature was removed before Chrome 79, it's not present in Edge
+      return { version_added: false };
     }
-  } else if (source == 'chrome') {
-    newData = originalData == undefined ? sourceData : originalData;
-
-    let chromeFalse =
-      sourceData.version_added === false ||
-      sourceData.version_removed !== undefined;
-    let chromeNull = sourceData.version_added === null;
 
-    if (originalData === undefined) {
-      newData.version_added = chromeFalse ? false : chromeNull ? null : '≤79';
-    } else {
-      if (!chromeFalse && !chromeNull) {
-        if (originalData.version_added == true) {
-          newData.version_added = '≤18';
-        } else {
-          if (
-            sourceData.version_added == true ||
-            Number(sourceData.version_added) <= 79
-          ) {
-            if (originalData.version_added == false) {
-              newData.version_added = '79';
-            } else if (originalData.version_added == null) {
-              newData.version_added = '≤79';
-            }
-          } else {
-            newData.version_added = sourceData.version_added;
-          }
-        }
-      } else if (chromeFalse) {
-        if (originalData.version_added && !originalData.version_removed) {
-          newData.version_removed = '79';
-        }
+    if (
+      typeof sourceData.version_added === 'string' &&
+      compareVersions.compare(sourceData.version_added, '79', '<=')
+    ) {
+      // If the feature was added before Chrome 79 but removed afterwards
+      if (typeof originalData.version_added == 'string') {
+        newData.version_added = originalData.version_added;
+      } else {
+        newData.version_added = '79';
       }
     }
-
-    let newNotes = combineNotes(
-      updateNotes(sourceData.notes, /Chrome/g, 'Edge'),
-      originalData.notes,
-    );
-
-    if (newNotes) {
-      newData.notes = newNotes;
+    if (originalData.version_added && !originalData.version_removed) {
+      newData.version_removed = '79';
+    }
+  } else if (chromeNull) {
+    newData.version_added = null;
+  } else {
+    if (sourceData.version_added === true) {
+      newData.version_added =
+        originalData.version_added === true ? '≤18' : true;
+    } else if (compareVersions.compare(sourceData.version_added, '79', '<=')) {
+      newData.version_added =
+        originalData.version_added === null ? '≤79' : '79';
+    } else {
+      newData.version_added = sourceData.version_added;
```

Comment:
This means for features introduced in Chrome 80 or later, the feature got introduced in the exact same Edge version!?

Reply:
Precisely, yep!

Reply:
Probably some concise code comments for each of the non-trivial branches would be helpful, even if those branches get removed eventually.

Reply:
Would it be alright to actually hold off on that for now?  These lines of code aren't new, just unindented from the existing lines.  (I'm also planning on following this up _immediately_ with removing usage of the original Edge data, so the comments would last a day or two.  :P)

Reply:
👍